src/aleph/sdk/client/abstract.py

```diff
@@ -168,25 +180,35 @@ async def get_messages(
         :param message_filter: Filter to apply to the messages
         :param ignore_invalid_messages: Ignore invalid messages (Default: True)
         :param invalid_messages_log_level: Log level to use for invalid messages (Default: logging.NOTSET)
+        :param verify_signatures: Whether to verify the signatures of the messages (Default: False)
```

Comment:
Would it be more readable in the imperative form ?
```suggestion
        :param verify_signatures: Verify the signatures of the messages (Default: False)
```